install/kubernetes/helm/istio/values.yaml

```diff
@@ -579,3 +579,8 @@ global:
   # This field is set to false by default, so 'helm template ...'
   # will ignore the helm test yaml files when generating the template
   enableHelmTest: false
+
+  # Configures DNS certificates provisioned through Chiron linked into Pilot.
+  # Te values in this file are all hard-coded; please ensure the namespaces
+  # in dnsNames are consistent with those of your services.
+  certificates: []
```

Comment:
Isn't there already this field somewhere for the Chiron stuff?

Reply:
In the istio/installer repository, https://github.com/istio/installer/pull/424 has this field as follows:
  certificates:
    - secretName: dns.istio-galley-service-account
      dnsNames: [istio-galley.istio-system.svc, istio-galley.istio-system]
    - secretName: dns.istio-sidecar-injector-service-account
      dnsNames: [istio-sidecar-injector.istio-system.svc, istio-sidecar-injector.istio-system]